src/dotnet-compile-fsc/Program.cs

```diff
@@ -168,11 +174,8 @@ public static int Run(string[] args)
                 allArgs.Add("--target:exe");
 
                 //HACK we need default.win32manifest for exe
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                {
-                    var win32manifestPath = Path.Combine(AppContext.BaseDirectory, "default.win32manifest");
-                    allArgs.Add($"--win32manifest:{win32manifestPath}");
-                }
+                var win32manifestPath = Path.Combine(fscExeDir, "..", "..", "runtimes", "any", "native", "default.win32manifest");
```

Comment:
@enricosada when dotnet-compile-fsc is a tool, win32manifest needs to be resolved relative to fsc.exe with the current package layout of FSharp.Compilers